src/components/Tree/TreeItem/TreeItem.tsx

```diff
@@ -73,6 +73,12 @@ export const TreeItem = memo(
         registerNodeChildren,
         unregisterNodeChildren,
         draggable: itemDraggable = true,
+        showDragHandlerOnHoverOnly = true,
+        dragHandlerPosition = 'LEFT',
```

Comment:
```suggestion
        dragHandlerPosition = 'left',
```
Let's relax and not scream